src/ensembl/src/content/app/browser/browserState.ts

```diff
@@ -25,6 +26,38 @@ export type CogList = {
   [key: string]: number;
 };
 
+export type RegionValidationError = {
+  error_code: string | null;
+  error_message: string | null;
+  is_valid: boolean;
+};
```

Comment:
why does a type that is called `error` have an is_valid field that is boolean (which means that it can be true)?

isn't it rather a ValidationResult or something?